packages/rum/src/domain/record/observers.ts

```diff
@@ -151,23 +152,26 @@ export function initMutationObserver(
   return startMutationObserver(cb, configuration, shadowRootsController, document)
 }
 
-function initMoveObserver(cb: MousemoveCallBack): ListenerHandler {
+export function initMoveObserver(cb: MousemoveCallBack): ListenerHandler {
   const { throttled: updatePosition } = throttle(
     monitor((event: MouseEvent | TouchEvent) => {
       const target = getEventTarget(event)
       if (hasSerializedNode(target)) {
-        const { clientX, clientY } = isTouchEvent(event) ? event.changedTouches[0] : event
+        const { x, y } = computeCoordinates(event)
+        if (!Number.isFinite(x) || !Number.isFinite(y)) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏move this check in `computeCoordinates` to avoit repeating it.

Reply:
So you would do something something like that?
```ts
const { x, y, areCoordinatesCorrect } = computeCoordinates(event)
if (areCoordinatesCorrect) {
```

Reply:
Maybe something like:
```ts
const coordinates = tryComputeCoordinates(event)
if (!coordinates) {
  return
}
// use coordinates without destructuring assignment
```